### PR TITLE
DA Sweeper Changes Two

### PR DESCRIPTION
**Pull request by Dala0**
This changes how asset_folder_sweeper.py works so it uses functions which sit in their own Python files. It also adds some more unit testing.

The purpose of this change is to allow testing of asset_folder_sweeper.py with Circle CI.

@fredex42

### CODE REVIEW

File: src/asset_folder_importer/tests/test_check_mime.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+        db = importer_db(__version__,hostname=self.dbhost,port=self.dbport,username=self.dbuser,
+                         password=self.dbpass,dbname=self.dbname)
+
+        filepath = "/etc/hosts"
+
+        (statinfo, mimetype) = check_mime(filepath, db)
+
+        self.assertEqual(mimetype, "text/plain")
+        self.assertNotEqual(statinfo, None)
```
Comment (fredex42) [inline src/asset_folder_importer/tests/test_check_mime.py]: can we get a couple more tests in here, e.g. an image or something?

Comment (Dala0) [inline src/asset_folder_importer/tests/test_check_mime.py]: Possibly. Any idea where an image file is always present on a UNIX like operating system? I went for /etc/hosts as it was likely to be present on any box.

Comment (Dala0) [inline src/asset_folder_importer/tests/test_check_mime.py]: I guess I could just put an image file in this and then attempt to load it. Not eligant, but it should work.

Comment (Dala0) [inline src/asset_folder_importer/tests/test_check_mime.py]: This has been added.

**Pull request merged**